web/vapour-style.json

```diff
@@ -254,47 +254,97 @@
       }
     },
     {
-      "id": "transport_ways_filtered_outline",
-      "type": "line",
+      "id": "line-intercity-copy",
+      "type": "symbol",
       "metadata": {},
       "source": "vapour_trail",
-      "source-layer": "route_way_members",
+      "source-layer": "ways",
+      "minzoom": 13,
       "filter": [
-        "==",
-        "rel_osm_id",
-        "dumb_filter"
+        "all",
+        [
+          ">",
+          "diameter",
+          3000
+        ]
       ],
       "layout": {
-        "line-cap": "round",
-        "line-join": "miter"
+        "text-field": "{rel_ref}",
+        "text-size": {
+          "stops": [
+            [
+              12,
+              6
+            ],
+            [
+              18,
+              12
+            ]
+          ]
+        },
+        "text-max-width": 2
       },
       "paint": {
-        "line-color": "#000",
-        "line-width": 6
+        "text-color": "rgba(255, 255, 255, 1)",
+        "text-halo-color": "#4898ff",
+        "text-halo-width": 5
       }
     },
     {
-      "id": "transport_ways_filtered",
+      "id": "line-urban-copy",
+      "type": "symbol",
+      "metadata": {},
+      "source": "vapour_trail",
+      "source-layer": "ways",
+      "minzoom": 14,
+      "filter": [
+        "all",
+        [
+          "<=",
+          "diameter",
+          3000
+        ]
+      ],
+      "layout": {
+        "text-field": "{rel_ref}",
+        "text-size": {
+          "stops": [
+            [
+              12,
+              6
+            ],
+            [
+              18,
+              12
+            ]
+          ]
+        },
+        "text-max-width": 2
+      },
+      "paint": {
+        "text-halo-color": "#4898ff",
+        "text-color": "rgba(255, 255, 255, 1)",
+        "text-halo-width": 5
```

Comment:
We really need a better picto for the bus stops : this one has no affordance, and it is really worse now because the shields of the lines  have the same color :thinking: 
![screenshot-2018-4-24 http localhost 1](https://user-images.githubusercontent.com/919962/39207819-4488f6f4-4801-11e8-81e8-7c872dd99c0a.png)

Reply:
I setup the shields on way, because the database layout only permit to do it in this way. But It should be better on stops.